Sources/Sentry/Public/SentrySpan.h

```diff
@@ -1,9 +1,97 @@
-#import <Foundation/Foundation.h>
+#import "SentrySpanContext.h"
+#import <Sentry/Sentry.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class SentryTransaction, SentrySpanId, SentryId, SentryHub;
+
 NS_SWIFT_NAME(Span)
-@protocol SentrySpan <NSObject>
+@interface SentrySpan : SentrySpanContext
+SENTRY_NO_INIT
+
+/**
+ * The timestamp of which the span ended.
+ */
+@property (nullable, nonatomic, strong) NSDate *timestamp;
+
+/**
+ * The start time of the span.
+ */
+@property (nullable, nonatomic, strong) NSDate *startTimestamp;
+
+/**
+ * An arbitrary mapping of additional metadata of the span.
+ */
+@property (nullable, readonly) NSDictionary<NSString *, id> *extras;
+
+/**
+ * Whether the span is finished.
+ */
+@property (readonly) BOOL isFinished;
```

Comment:
L: on Java it's not a property, we do `timestamp != null` means `finished`, not against the property though

Reply:
.NET is a property but the getter actually does the same as Java
https://github.com/getsentry/sentry-dotnet/blob/39680350f97cc4c6901f7a0f56d7ad7f2f2aa46a/src/Sentry/Span.cs#L37

Reply:
this is a calculated property as .NET